gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -912,4 +912,18 @@
   public static final String GIT_CONFIG_MONITOR_BRANCH_NAME = GIT_CONFIG_MONITOR_PREFIX + "branchName";
   public static final String DEFAULT_GIT_CONFIG_MONITOR_BRANCH_NAME = "master";
   public static final int DEFAULT_GIT_CONFIG_MONITOR_POLLING_INTERVAL = 60;
+
+  /**
+   * Configuration related to the git flow graph monitoring service
+   */
+  public static final String GIT_FLOWGRAPH_MONITOR_PREFIX = "gitFlowGraphMonitor.";
```

Comment:
This set of keys is a copy of the ones above. Can you refactor the logic to share the keys with the defaults encapsulated in the implementations? This allows more of the initialization code to be shared.